bin/improver-wind-downscaling

```diff
@@ -0,0 +1,116 @@
+                             ' file. Any units can be supplied.')
+    parser.add_argument('silhouette_roughness_filepath', metavar='AOS_FILE',
+                        help='Location of model silhouette roughness file. '
+                             'Units of field: dimensionless')
+    parser.add_argument('sigma_filepath', metavar='SIGMA_FILE',
+                        help='Location of standard deviation of model '
+                             'orography height file. Units of field: m')
+    parser.add_argument('target_orog_filepath',
+                        metavar='TARGET_OROGRAPHY_FILE',
+                        help='Location of target orography file to downscale'
+                             ' fields to.'
+                             'Units of field: m')
+    parser.add_argument('standard_orog_filepath',
+                        metavar='STANDARD_OROGRAPHY_FILE',
+                        help='Location of orography on standard grid file '
+                             '(interpolated model orography.'
+                             ' Units of field: m')
+    parser.add_argument('model_resolution', metavar='MODEL_RESOLUTION',
+                        help='Original resolution of model orography (before'
+                             ' interpolation to standard grid).'
+                             ' Units of field: m')
+    parser.add_argument('output_filepath', metavar='OUTPUT_FILE',
+                        help='The output path for the processed NetCDF')
+    parser.add_argument('--height_levels_filepath',
+                        metavar='HEIGHT_LEVELS_FILE',
+                        help='Location of file containing height levels '
+                             'coincident with wind speed field.')
+    parser.add_argument('--veg_roughness_filepath',
+                        metavar='VEGETATIVE_ROUGHNESS_LENGTH_FILE',
+                        help='Location of vegetative roughness length file.'
+                             ' Units of field: m')
+    args = parser.parse_args()
+    wind_speed = iris.load_cube(args.wind_speed_filepath)
+    silhouette_roughness_filepath = iris.load_cube(
+        args.silhouette_roughness_filepath)
+    sigma = iris.load_cube(args.sigma_filepath)
+    target_orog = iris.load_cube(args.target_orog_filepath)
+    standard_orog = iris.load_cube(args.standard_orog_filepath)
+    if args.height_levels_filepath:
+        height_levels = iris.load_cube(args.height_levels_filepath)
+    else:
+        height_levels = None
+    if args.veg_roughness_filepath:
+        veg_roughness_cube = iris.load_cube(args.veg_roughness_filepath)
+    else:
+        veg_roughness_cube = None
+    try:
+        realization_coord = wind_speed.coord('realization')
+        wind_speed_iterator = wind_speed.slices_over('realization')
+    except CoordinateNotFoundError:
+        wind_speed_iterator = [wind_speed]
+    wind_speed_list = iris.cube.CubeList()
+    for wind_speed_slice in wind_speed_iterator:
```

Comment:
Nice, I like this - as discussed, we can multiprocess this out in future if we need to.